CMakeLists.txt

```diff
@@ -37,6 +37,8 @@ endif()
 # Extension modules
 add_subdirectory(AutoscoperM)
 add_subdirectory(TrackingEvaluation)
+add_subdirectory(calculateDataIntensityDensity)
```

Comment:
For consistency, I suggest to have all module directories start with an upper case letter